R/tableone.R

```diff
@@ -22,11 +23,12 @@ tableone <- function(
     # abbreviations = "",
     # variable_definitions = "",
     strata = NULL,
+    overall = T,
```

Comment:
Will this not lead to a weird behaviour when strata = NULL => an overall stratum is added. Do we still need an overall calculation?